source/models/database.py

```diff
@@ -34,9 +36,8 @@ def __serialize_tables(self):
             table.serialize()
 
     def set(self, table_name, row):
-        # table = Table(table_name)
-        # table.set(row)
-        pass
+        table = Table(self, table_name = table_name)
```

Comment:
You already have a map of tables. You can get the table object from it. You don’t need to create a new one

Reply:
ok done